README.md

```diff
@@ -110,7 +112,7 @@ module "observability_instances" {
             {
                 action = "send"
                 targets = [{
-                    id = module.observability_instances.metrics_router_targets["my-mr-target].id
+                    id = module.observability_instances.metrics_router_targets["my-mr-target"].id
```

Comment:
Not sure about this name either... but my-mr-target is probably in the code elsewhere.

Reply:
`my-mr-target` was in the readme as well as in the example code so removed it.